src/view-components/widget/widget.component.tsx

```diff
@@ -1,45 +1,60 @@
 import React, { FunctionComponent, useContext } from "react";
-import { AppPropsContext } from "../../app-props-context";
+//@ts-ignore The present types for single-spa-react are not updated yet for 2.9 which has SingleSpaContext
+import s, { SingleSpaContext } from "single-spa-react";
```

Comment:
what's `s`?

Reply:
s is single-spa-react but we're not use it so I have removed it.